config/vale/styles/config/vocabularies/FusionAuth/accept.txt

```diff
@@ -1505,6 +1505,8 @@ ZanoCloud
 Zanozin
 Zapier
 Zendesk
+Ctrl
```

Comment:
We have switched to making these alphabetical, could you please update accordingly?